test/integration/scenarios/network/peers.js

```diff
@@ -126,14 +73,14 @@ module.exports = function(params) {
 
 		describe('forging', () => {
 			before(done => {
-				// Expect some blocks to forge after 30 seconds
-				var timesToCheckNetworkStatus = 30;
-				var timesNetworkStatusChecked = 0;
-				var checkNetworkStatusInterval = 1000;
-
-				var checkingInterval = setInterval(() => {
-					getNodesStatus((err, data) => {
-						var { networkMaxAvgHeight } = data;
+				// Expect some blocks to be forged after 30 seconds
```

Comment:
Some? We don't care how many?

Reply:
Yes, the consensus might be lacking so potentially a few might be skipped. The test still looks brittle :) Reimplementing it with the proper description would be also outside of scope of this issue.